src/components/MultiHopTrade/components/TradeInput/components/TradeQuotes/TradeQuotes.tsx

```diff
@@ -93,14 +156,71 @@ export const TradeQuotes: React.FC<TradeQuotesProps> = memo(({ isLoading, onBack
     isTradeQuoteRequestAborted,
     bestQuoteData?.quote,
     bestQuoteData?.inputOutputRatio,
-    tradeQuoteDisplayCache,
+    validTradeQuotesDisplayCache,
     activeQuoteMeta,
     isTradeQuoteApiQueryPending,
     isLoading,
     isSwapperQuoteAvailable,
     onBack,
   ])
 
+  const invalidQuotes = useMemo(() => {
+    if (isTradeQuoteRequestAborted) {
+      return []
+    }
+
+    const bestTotalReceiveAmountCryptoPrecision = bestQuoteData?.quote
+      ? getBuyAmountAfterFeesCryptoPrecision({
+          quote: bestQuoteData.quote,
+        })
+      : undefined
+
+    return invalidTradeQuotesDisplayCache.map((quoteData, i) => {
+      const { swapperName, id, errors, isStale } = quoteData
+
+      const isActive = activeQuoteMeta !== undefined && activeQuoteMeta.identifier === id
+
+      return (
+        <TradeQuote
+          isActive={isActive}
+          isLoading={Boolean(isTradeQuoteApiQueryPending[quoteData.swapperName]) && isStale}
+          isRefetching={
+            isLoading ||
+            isTradeQuoteApiQueryPending[quoteData.swapperName] ||
+            !isSwapperQuoteAvailable[swapperName]
+          }
```

Comment:
question: If the swapper is refetching, would it be better to move this out of the "invalid" group so the user can see loading state? Otherwise wouldn't it be hidden here until a quoteis valid?

Reply:
I would like to avoid as we would have too much noises where 90% of the time the quote will stay invalid (no supported route for this trading pair, swapper not compatible... refetching won't change anything in most of the cases)

Happy to discuss this if you feel like it's weird at runtime but I found it fine already